### PR TITLE
Index

### PR DESCRIPTION
**Pull request by saugusth**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,14 +1,79 @@
...
-<html lang="en">
+
+<!doctype html>
+<html lang="en" class="h-100">
   <head>
     <meta charset="utf-8">
-    <title></title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="">
+    <meta name="author" content="Mark Otto, Jacob Thornton, and Bootstrap contributors">
```
Comment (sahil-at-home1) [inline src/index.html]: Who are these?

File: src/index.html [hunk 2]
```diff
@@ -1,14 +1,79 @@
...
+        </ul>
+      </div>
+    </div>
+  </nav>
+</header>
+
+<!-- Begin page content -->
+<main class="flex-shrink-0">
+  <div class="top-container"> <!--top-container is the top container-->
+    <div class="skill-share"> <!--skill-share is the style for h1 and the img-->
```
Comment (sahil-at-home1) [inline src/index.html:49]: why is the class called skill-share?

File: src/index.html (hunk 1 above)
Comment (nickwanninger) [reply src/index.html]: Those are the bootstrap developers. probably delete that (or change it to `IIT IPRO Students`

File: src/index.html [hunk 3]
```diff
@@ -1,14 +1,79 @@
...
+
+<!doctype html>
+<html lang="en" class="h-100">
   <head>
     <meta charset="utf-8">
-    <title></title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="">
+    <meta name="author" content="Mark Otto, Jacob Thornton, and Bootstrap contributors">
+    <meta name="generator" content="Hugo 0.80.0">
```
Comment (sahil-at-home1) [inline src/index.html]: what is the Hugo generator?

Comment (nickwanninger) [reply src/index.html]: https://gohugo.io/

( I still get notifications on this repo lol )

**Review by sahil-at-home1 — COMMENTED** [review]
Looks good so far. I like the index.html and the bootstrap containers. The navbar code should be good as well. I left some other comments about some things. We should discuss them today in the meeting.

File: src/index.html (hunk 3 above)
Comment (sahil-at-home1) [reply src/index.html]: thanks lol

File: src/index.html (hunk 2 above)
Comment (saugusth) [reply src/index.html:49]: i was using a tutorial to refresh my CSS and they call it that

**Comment by ilopez5** [comment]
Yeah don't merge this until we discuss it this evening, looks interesting though 👍🏽

**Pull request closed without merging**